pkgs/top-level/all-packages.nix

```
@@ -7738,6 +7738,10 @@ with pkgs;
 
   zoxide = callPackage ../tools/misc/zoxide { };
 
+  zotify = callPackage ../by-name/zo/zotify/package.nix {
+    inherit fetchFromGitHub python3Packages protobuf_21 lib;
+  };
+
```

callPackage will do all of this for you
```suggestion
  zotify = callPackage ../by-name/zo/zotify/package.nix { };

```

---

CI fails when I do that

---

Oh that's right, when using the pkgs/by-name hierarchy you can remove any entry from all-packages.nix altogether unless it calls callPackage with extra attributes.

---

Ah, so do I add the attributes back?